go/pkg/yolosvc/db.go

```diff
@@ -80,14 +51,13 @@ func (svc *service) saveBatch(ctx context.Context, batch *yolopb.Batch) error {
 	return nil
 }
 
-func lastBuildCreatedTime(ctx context.Context, db *gorm.DB, driver yolopb.Driver) (time.Time, error) {
-	build := yolopb.Build{
-		Driver: driver,
-	}
-	err := db.Order("finished_at desc").Where(&build).Select("finished_at").First(&build).Error
+func lastBuildCreatedTime(ctx context.Context, store yolostore.Store, driver yolopb.Driver) (time.Time, error) {
+
+	build, err := store.GetLastBuild()
 	if err != nil {
 		return time.Time{}, err
 	}
+	build.Driver = driver
```

Comment:
previous implementation ->
* filtering on a specific driver"

new implementation ->
* no driver filtering
* overriding the response driver